cytominer_eval/tests/test_utils/test_availability_utils.py

```diff
@@ -33,3 +38,44 @@ def test_get_available_eval_metrics():
 def test_get_available_similarity_metrics():
     expected_result = ["pearson", "kendall", "spearman"]
     assert expected_result == get_available_similarity_metrics()
+
+
+def test_get_available_distribution_compare_methods():
+    expected_result = ["zscore"]
+    assert expected_result == get_available_distribution_compare_methods()
+
+
+def test_check_eval_metric():
+    with pytest.raises(AssertionError) as ae:
+        output = check_eval_metric(eval_metric="NOT SUPPORTED")
+    assert "ot supported. Select one of" in str(ae.value)
+
+
+def test_check_replicate_summary_method():
+    # Pass
```

Comment:
is this part of the documentation? if so, its a bit short. 
Delete or expand for the following three functions